MADE.App.Views.Controls.HeaderedTextBlock/HeaderedTextBlock.Windows.cs

```diff
@@ -0,0 +1,188 @@
+    public class HeaderedTextBlock : Control, IHeaderedTextBlock
+    {
+        /// <summary>
+        /// Defines the dependency property for the <see cref="Header"/> value.
+        /// </summary>
+        public static readonly DependencyProperty HeaderProperty = DependencyProperty.Register(
+            nameof(Header),
+            typeof(string),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(null, (d, e) => ((HeaderedTextBlock)d).UpdateVisibility()));
+
+        /// <summary>
+        /// Defines the dependency property for the <see cref="HeaderStyle"/> value.
+        /// </summary>
+        public static readonly DependencyProperty HeaderStyleProperty = DependencyProperty.Register(
+            nameof(HeaderStyle),
+            typeof(Style),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(default(Style)));
+
+        /// <summary>
+        /// Defines the dependency property for the <see cref="Orientation"/> value.
+        /// </summary>
+        public static readonly DependencyProperty OrientationProperty = DependencyProperty.Register(
+            nameof(Orientation),
+            typeof(Orientation),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(Orientation.Vertical, (d, e) => ((HeaderedTextBlock)d).UpdateOrientation()));
+
+        /// <summary>
+        /// Defines the dependency property for the <see cref="Text"/> value.
+        /// </summary>
+        public static readonly DependencyProperty TextProperty = DependencyProperty.Register(
+            nameof(Text),
+            typeof(string),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(null, (d, e) => ((HeaderedTextBlock)d).UpdateVisibility()));
+
+        /// <summary>
+        /// Defines the dependency property for the <see cref="TextStyle"/> value.
+        /// </summary>
+        public static readonly DependencyProperty TextStyleProperty = DependencyProperty.Register(
+            nameof(TextStyle),
+            typeof(Style),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(default(Style)));
+
+        /// <summary>
+        /// Defines the dependency property for the <see cref="HideIfNullOrWhiteSpace"/> value.
+        /// </summary>
+        public static readonly DependencyProperty HideIfNullOrWhiteSpaceProperty = DependencyProperty.Register(
+            nameof(HideIfNullOrWhiteSpace),
+            typeof(bool),
+            typeof(HeaderedTextBlock),
+            new PropertyMetadata(false, (d, e) => ((HeaderedTextBlock)d).UpdateVisibility()));
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HeaderedTextBlock"/> class.
+        /// </summary>
+        public HeaderedTextBlock()
+        {
+            this.DefaultStyleKey = typeof(HeaderedTextBlock);
+        }
+
+        public TextBlock HeaderTextBlock { get; private set; }
```

Comment:
Can we make sure that the way these controls can be accessed across the platforms is similar with the naming of the control? Noticed that these are different to the ones in the Android project.

Reply:
I will make sure this is fixed